### PR TITLE
Integrated features

### PR DESCRIPTION
**Pull request by alan-wu**
This branch include the changes from @agarny [simulation](https://github.com/agarny/mapintegratedvuer/tree/simulation) branch, @alan-wu [splitscreen](https://github.com/alan-wu/mapintegratedvuer/tree/split-screen) branch and various map sidebar changes.

### CODE REVIEW

File: src/components/species-map.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
 export default {
```
Comment (agarny) [inline src/components/species-map.js:1]: I don't believe this file is needed anymore.

**Review by agarny — COMMENTED** [review]
Have just merged in your `integrated-features` branch in my copy of `mapintegratedvuer` and it all works fine for me. The only thing I noticed is that JS file which shouldn't be needed anymore.

File: src/components/species-map.js (hunk 1 above)
Comment (alan-wu) [reply src/components/species-map.js]: Sure, it is now removed.

**Review by agarny — APPROVED** [review]
From what I can tell, it looks fine to me.

**Review by Tehsurfer — APPROVED** [review]
The changes look good to me! 

I would let the user know that this number is the dataset id though, I was a bit confused what it was at first:

image

I also thought it could be better space usage if the dailogue keys are hidden based off of a media query?
image

Not really a mapintegratedvuer problem, but also looks like simulation parameters have some sizing issues at small heights
image

**Comment by agarny** [comment]
> The changes look good to me!
> 
> I would let the user know that this number is the dataset id though, I was a bit confused what it was at first:

I think the idea (as a result of the UI/UX testing) was (obviously?) to distinguish between the different simulations since before we were just mentioning "Simulation". Still, it's not great, I agree.

> I also thought it could be better space usage if the dailogue keys are hidden based off of a media query?

+1

> Not really a mapintegratedvuer problem, but also looks like simulation parameters have some sizing issues at small heights

What do you mean?

**Comment by alan-wu** [comment]
It is actually hard to fit in the word dataset in the title, that was the case before but it would not be easy to see the actual dateset id with more words in the title.

Regarding hidding the controls - something we should consider doing next sprint.

**Pull request merged**